charts/bitwardenrs/1.1.1/values.yaml

```diff
@@ -0,0 +1,178 @@
+#   smtp:
+#     enabled: false
+#     # SMTP hostname, required if SMTP is enabled
+#     host: ""
+#     # SMTP sender e-mail address, required if SMTP is enabled
+#     from: ""
+#     # SMTP sender name, defaults to 'Bitwarden_RS'
+#     fromName: ""
+#     # Enable SSL connection
+#     ssl: true
+#     # SMTP port
+#     port: 587
+#     # SMTP username
+#     user: ""
+#     # SMTP password. Required is user is specified, ignored if no user provided
+#     password: ""
+#     # Use existing secret for SMTP authentication
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       userKey: ""
+#       passwordKey: ""
+#   # Enable Yubikey 2FA: https://github.com/dani-garcia/bitwarden_rs/wiki/Enabling-Yubikey-OTP-authentication
+#   yubico:
+#     enabled: false
+#     # OTP verification server. Will use the default YubiCloud servers if not specified
+#     server: ""
+#     # API Client ID for OTP server. Ignored if existingSecret is provided.
+#     clientId: ""
+#     # API Secret Key for OTP server. Required if clientId is specified, ignored when using existingSecret.
+#     secretKey: ""
+#     # Use existing secret for API keys
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       clientIdKey: ""
+#       secretKeyKey: ""
+
+# env: {}
+# # If you plan to run the WebUI on a port other than port 80, specify that here:
+# # For example, if running the container as a non-root user.
+# #  ROCKET_PORT: "80"
+
+# persistence:
+#   type: statefulset
+#   enabled: false
+#   size: 1Gi
+#   accessMode: ReadWriteOnce
+#   ## Persistent Volume storage class
+#   # storageClass: "-"
+#   ## Use existing Persistent Volume Claim
+#   # existingClaim:
+
+# serviceAccount:
+#   # Specifies whether a service account should be created
+#   create: true
+#   # Annotations to add to the service account
+#   annotations: {}
+#   # The name of the service account to use.
+#   # If not set and create is true, a name is generated using the fullname template
+#   name: ""
+
+# podAnnotations: {}
+
+# podSecurityContext: {}
+#   # fsGroup: 2000
+
+# securityContext: {}
+#   # capabilities:
+#   #   drop:
+#   #   - ALL
+#   # readOnlyRootFilesystem: true
+#   # runAsNonRoot: true
+#   # runAsUser: 1000
+
+service:
+  port:
+    port: 80
+#   ## Provide any additional annotations which may be required. This can be used to
+#   ## set the LoadBalancer service type to internal only.
+#   ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+#   ##
+#   annotations: {}
+#   labels: {}
+#   additionalSpec: {}
+
+# ingress:
+#   enabled: false
+#   annotations: {}
+#     # kubernetes.io/ingress.class: nginx
+#     # kubernetes.io/tls-acme: "true"
+#   hosts:
+#     - host: chart-example.local
+#       paths: []
+#   tls: []
+#   #  - secretName: chart-example-tls
+#   #    hosts:
+#   #      - chart-example.local
```

Comment:
if k8s-at-home thinks this is the correct way of setting up ingress for bitwarden, someone needs a spanking. It's not, so please do not base you ingress on this ;-)